bin/pgenv

```diff
@@ -1417,17 +1427,29 @@ EOF
         fi
 
         title="PostgreSQL version $v"
+        configuration_file=$( pgenv_configuration_file_name $v )
 
         case $action in
             show)
                 pgenv_configuration_dump_or_exit "$v" "$title" ;;
-            write)
+            init)
+                # check that the file does not exist, so
+                # that the program does not re-initialize it
+                if [ -f "$configuration_file" ]; then
+                    echo "Configuration file [$configuration_file] for version [$v] already exists"
```

Comment:
I suggest omitting `for version [$v]`, since IIUC it will sometimes be `for version [default]`, which doesn't make sense, and the version Is in `$configuration_file` anyway, is it not?

Reply:
> I suggest omitting `for version [$v]`, since IIUC it will sometimes be `for version [default]`, which doesn't make sense, and the version Is in `$configuration_file` anyway, is it not?

Fixed the warning message and added a debug message to explicitly telle the version.